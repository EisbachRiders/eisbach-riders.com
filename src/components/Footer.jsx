import React, { Fragment } from "react"
import { useTranslation } from "react-i18next"
import { Link } from "gatsby-theme-material-ui"
import clsx from "clsx"
import { makeStyles } from "@material-ui/styles"
import IconButton from "@material-ui/core/IconButton"
import Hidden from "@material-ui/core/Hidden"
import FacebookIcon from "@material-ui/icons/Facebook"
import InstagramIcon from "@material-ui/icons/Instagram"
import PinterestIcon from "@material-ui/icons/Pinterest"
import QuestionAnswerIcon from "@material-ui/icons/QuestionAnswer"
import ReplyIcon from "@material-ui/icons/Reply"
import LocalShippingIcon from "@material-ui/icons/LocalShipping"
import Newsletter from "./newsletter/Newsletter"
import Container from "./ui/Container"
import Contact from "./Contact"
import fullLogo from "../assets/logos/ER_full_black.svg"

const useStyles = makeStyles((theme) => ({
  footer: {
    background: theme.color.white,
  },
  container: {
    justifyContent: "center",
    [theme.breakpoints.up("sm")]: {
      justifyContent: "space-between",
    },
  },
  title: {
    textTransform: "lowercase",
    fontSize: 16,
    letterSpacing: 3,
    marginBottom: 15,
    [theme.breakpoints.up("md")]: {
      fontSize: 24,
    },
  },
  text: {
    textTransform: "capitalize",
    color: theme.color.black,
    letterSpacing: 2,
    marginBottom: 15,
    fontSize: 12,
    [theme.breakpoints.up("md")]: {
      fontSize: 16,
    },
  },
  copyrightContainer: {
    display: "flex",
    flexWrap: "wrap",
    alignItems: "center",
    color: theme.color.white,
    paddingTop: 10,
    [theme.breakpoints.up("sm")]: {
      paddingTop: 0,
    },
  },
  textSmall: {
    color: theme.color.white,
    fontSize: 12,
    textTransform: "capitalize",
    letterSpacing: 2,
    fontWeight: 500,
    padding: "0 10px",
    width: "49%",
    textAlign: "center",
    [theme.breakpoints.up("sm")]: {
      width: "fit-content",
    },
  },
  link: {
    "&:hover": {
      textDecoration: "none",
      color: theme.palette.primary.main,
    },
  },
  img: {
    width: 150,
    marginBottom: 30,
  },
  linksContainer: {
    display: "flex",
    justifyContent: "space-between",
  },
  list: {
    display: "flex",
    flexDirection: "column",
    margin: "0 15px",
    [theme.breakpoints.up("md")]: {
      margin: "0 45px",
    },
  },
  imgContainer: {
    display: "flex",
    flexDirection: "column",
    [theme.breakpoints.up("md")]: {
      height: "100%",
    },
  },
  socialContainer: {
    display: "flex",
    justifyContent: "space-around",
  },
  iconSocial: {
    color: theme.color.black,
    width: 22,
    height: 22,
    "&:hover": {
      color: theme.palette.primary.main,
    },
  },
  iconContainer: {
    display: "flex",
    flexBasis: "100%",
    [theme.breakpoints.up("sm")]: {
      flexBasis: "30%",
    },
    [theme.breakpoints.up("sm")]: {
      flexBasis: "25%",
      justifyContent: "space-around",
    },
  },
  icon: {
    width: 32,
    height: 32,
    marginRight: 30,
    [theme.breakpoints.up("sm")]: {
      width: 24,
      height: 24,
    },
    [theme.breakpoints.up("md")]: {
      width: 32,
      height: 32,
    },
    [theme.breakpoints.up("lg")]: {
      marginRight: 30,
      width: 42,
      height: 42,
    },
  },
  feature: {
    fontSize: 16,
    letterSpacing: 3,
    marginTop: 0,
    marginBottom: 5,
    [theme.breakpoints.up("sm")]: {
      fontSize: 13,
    },
    [theme.breakpoints.up("lg")]: {
      fontSize: 22,
    },
  },
  featureText: {
    marginTop: 0,
    color: theme.color.textLight,
    fontSize: 14,
    [theme.breakpoints.up("sm")]: {
      fontSize: 12,
    },
    [theme.breakpoints.up("md")]: {
      fontSize: 12,
    },
  },
  copyright: {
    width: "100%",
    [theme.breakpoints.up("sm")]: {
      width: "fit-content",
    },
  },
}))

function Footer() {
  const classes = useStyles()
  const { t } = useTranslation()
  const blog = ["lifestyle", "travel", "gear", "community"]
  const shop = ["rapid", "fins", "sup", "leashes", "accessories", "apparel"]
  const customerService = ["faq", "contact", "shipping", "returns"]
  const features = [
    {
      title: "emailSupport",
      icon: <QuestionAnswerIcon className={classes.icon} />,
    },
    {
      title: "30dayReturns",
      icon: <ReplyIcon className={classes.icon} />,
    },
    {
      title: "freeShipping",
      icon: <LocalShippingIcon className={classes.icon} />,
    },
  ]
  return (
    <footer className={classes.footer}>
      <Container justifyContent="spaceAround">
        {features.map((elem, idx) => (
          <div className={classes.iconContainer} key={`feature${idx}`}>
            {elem.icon}
            <div>
              <p className={classes.feature}>{t(`footer.${elem.title}`)}</p>
              <p className={classes.featureText}>
                {t(`footer.${elem.title}Text`)}
              </p>
            </div>
          </div>
        ))}
      </Container>

      <Newsletter />

      <Container alignItems="flexStart" className={classes.container}>
        <div className={classes.imgContainer}>
          <img src={fullLogo} alt="logo" className={classes.img} />
          <div className={classes.socialContainer}>
            <IconButton
              href="https://www.facebook.com/EisbachRiders/"
              aria-label="facebook"
              size="small"
            >
              <FacebookIcon className={classes.iconSocial} />
            </IconButton>
            <IconButton
              href="https://www.instagram.com/eisbachriders/"
              aria-label="instagram"
              size="small"
            >
              <InstagramIcon className={classes.iconSocial} />
            </IconButton>
            <IconButton
              href="https://www.pinterest.com/eisbachriders/"
              aria-label="pinterest"
              size="small"
            >
              <PinterestIcon className={classes.iconSocial} />
            </IconButton>
          </div>
        </div>
        <Hidden xsDown>
          <div className={classes.linksContainer}>
            <div className={classes.list}>
              <p className={classes.title}>{t("links.blog")}</p>
              {blog.map((elem, idx) => (
                <Link
                  key={`blog${idx}`}
                  to={`/tags/${elem}`}
                  className={clsx(classes.text, classes.link)}
                >
                  {t(`links.${elem}`)}
                </Link>
              ))}
            </div>
            <div className={classes.list}>
              <p className={classes.title}>{t("links.shop")}</p>
              {shop.map((elem, idx) => (
                <Link
                  key={`shop${idx}`}
                  to={`/${elem}`}
                  className={clsx(classes.text, classes.link)}
                >
                  {t(`links.${elem}`)}
                </Link>
              ))}
            </div>
            <div className={classes.list}>
              <p className={classes.title}>{t("links.customerService")}</p>
              {customerService.map((elem, idx) => (
                <Fragment key={`customerService${idx}`}>
                  {elem === "contact" ? (
                    <Contact variant="link" />
                  ) : (
                    <Link
                      to={`/${elem}`}
                      className={clsx(classes.text, classes.link)}
                    >
                      {t(`links.${elem}`)}
                    </Link>
                  )}
                </Fragment>
              ))}
            </div>
          </div>
        </Hidden>
      </Container>

      <Container
        justifyContent="spaceBetween"
        background="black"
        alignItems="center"
        padding="none"
      >
        <div className={classes.copyrightContainer}>
          <Link
            to="/dataProtection"
            className={clsx(classes.textSmall, classes.link)}
          >
            {t("links.data")}
          </Link>
          |
          <Link
            to="/termsAndConditions"
            className={clsx(classes.textSmall, classes.link)}
          >
            {t("links.terms")}
          </Link>
          <Hidden smDown>|</Hidden>
          <Link to="/imprint" className={clsx(classes.textSmall, classes.link)}>
            {t("links.imprint")}
          </Link>
          |
          <Link to="/credit" className={clsx(classes.textSmall, classes.link)}>
            {t("links.credit")}
          </Link>
        </div>
        <p className={clsx(classes.textSmall, classes.copyright)}>
          &copy; Eisbach Riders
        </p>
      </Container>
    </footer>
  )
}

export default Footer
