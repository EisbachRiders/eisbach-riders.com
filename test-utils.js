import React from 'react'
import { render } from '@testing-library/react'
import { I18nextProvider } from 'react-i18next'
import i18n from './src/i18n/i18n'
import { ThemeProvider } from '@material-ui/styles'
import theme from './src/theme'

const AllTheProviders = ({ children }) => {
  return (
    <ThemeProvider theme={theme}>
      <I18nextProvider i18n={i18n}>{children}</I18nextProvider>
    </ThemeProvider>
  )
}

const customRender = (ui, options) =>
  render(ui, { wrapper: AllTheProviders, ...options })

// re-export everything
export * from '@testing-library/react'

// override render method
export { customRender as render }
