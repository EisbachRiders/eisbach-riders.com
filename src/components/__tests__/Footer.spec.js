import React from 'react'
import { render, fireEvent } from '../../../test-utils'
import Footer from '../Footer'
import CircularProgress from '@material-ui/core/CircularProgress'
// import { handleSubmit as mockSavePost } from '../Footer'

// jest.mock('../handleSubmit', () => {
//   return {
//     handleSubmit: jest.fn(() => Promise.resolve),
//   }
// })

// afterEach(() => {
//   mockSavePost.mockClear()
// })

describe('Footer', () => {
  it('renders correctly', () => {
    const component = render(<Footer />)
    expect(component).toMatchSnapshot()
  })
  // it('submits button', () => {
  //   const { getByTestId, getByPlaceholderText } = render(<Footer />)
  //   const fakePost = {
  //     footer_name: 'mockName',
  //     footer_email: 'mockEmail@test.com',
  //     footer_message: 'mockMessage',
  //   }
  //   getByPlaceholderText('common.name').value = fakePost.footer_name
  //   getByPlaceholderText('common.email').value = fakePost.footer_email
  //   getByPlaceholderText('footer.message').value = fakePost.footer_message
  //   const button = getByTestId('footer_submit')
  //   fireEvent.click(button)
  //   expect(<CircularProgress />).toBeDefined()
  //   // expect(mockSavePost).toHaveBeenCalledTimes(1)
  //   // expect(mockSavePost).toHaveBeenCalledWith(fakePost)
  // })
})
