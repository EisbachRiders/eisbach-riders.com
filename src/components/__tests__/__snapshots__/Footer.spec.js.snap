// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="Hook-root-4c7lha"
      >
        <div
          class="Hook-container-1tixbb2"
        >
          <div
            class="Hook-flexItem-16n2577"
          >
            <form
              autocomplete="off"
              novalidate=""
            >
              <p
                class="MuiTypography-root-8 MuiTypography-body2-9 Hook-textHeading-3xpc43"
              >
                footer.message
              </p>
              <div
                class="Hook-inputContainer-11ncpjx"
              >
                <div
                  class="MuiFormControl-root-39 MuiFormControl-marginNormal-40 MuiTextField-root-38 Hook-textField-xmu9w"
                >
                  <label
                    class="MuiFormLabel-root-55 MuiFormLabel-required-60 MuiInputLabel-required-47 MuiInputLabel-root-43 MuiInputLabel-formControl-49 MuiInputLabel-animated-52 Hook-inputLabel-1dirayr"
                    data-shrink="false"
                    for="footer_name"
                  >
                    common.name
                    <span
                      class="MuiFormLabel-asterisk-61 MuiInputLabel-asterisk-48"
                    >
                       *
                    </span>
                  </label>
                  <div
                    class="MuiInputBase-root-75 MuiInput-root-62 MuiInputBase-formControl-76 MuiInput-formControl-63 Hook-input-2r9hv1"
                  >
                    <input
                      aria-invalid="false"
                      class="MuiInputBase-input-85 MuiInput-input-70 Hook-textArea-1fn7y44"
                      id="footer_name"
                      placeholder="common.name"
                      required=""
                      type="text"
                      value=""
                    />
                  </div>
                </div>
                <div
                  class="MuiFormControl-root-39 MuiFormControl-marginNormal-40 MuiTextField-root-38 Hook-textField-xmu9w"
                >
                  <label
                    class="MuiFormLabel-root-55 MuiFormLabel-required-60 MuiInputLabel-required-47 MuiInputLabel-root-43 MuiInputLabel-formControl-49 MuiInputLabel-animated-52 Hook-inputLabel-1dirayr"
                    data-shrink="false"
                    for="footer_email"
                  >
                    common.email
                    <span
                      class="MuiFormLabel-asterisk-61 MuiInputLabel-asterisk-48"
                    >
                       *
                    </span>
                  </label>
                  <div
                    class="MuiInputBase-root-75 MuiInput-root-62 MuiInputBase-formControl-76 MuiInput-formControl-63 Hook-input-2r9hv1"
                  >
                    <input
                      aria-invalid="false"
                      class="MuiInputBase-input-85 MuiInput-input-70 Hook-textArea-1fn7y44 MuiInputBase-inputType-88 MuiInput-inputType-73"
                      id="footer_email"
                      placeholder="common.email"
                      required=""
                      type="email"
                      value=""
                    />
                  </div>
                </div>
              </div>
              <div
                class="MuiFormControl-root-39 MuiFormControl-marginNormal-40 MuiFormControl-fullWidth-42 MuiTextField-root-38"
              >
                <label
                  class="MuiFormLabel-root-55 MuiFormLabel-required-60 MuiInputLabel-required-47 MuiInputLabel-root-43 MuiInputLabel-formControl-49 MuiInputLabel-animated-52 Hook-inputLabel-1dirayr"
                  data-shrink="false"
                  for="footer_message"
                >
                  footer.message
                  <span
                    class="MuiFormLabel-asterisk-61 MuiInputLabel-asterisk-48"
                  >
                     *
                  </span>
                </label>
                <div
                  class="MuiInputBase-root-75 MuiInput-root-62 MuiInputBase-fullWidth-84 MuiInput-fullWidth-69 MuiInputBase-formControl-76 MuiInput-formControl-63 MuiInputBase-multiline-83 MuiInput-multiline-68 Hook-input-2r9hv1"
                >
                  <textarea
                    aria-invalid="false"
                    class="MuiInputBase-input-85 MuiInput-input-70 Hook-textArea-1fn7y44 MuiInputBase-inputMultiline-87 MuiInput-inputMultiline-72"
                    id="footer_message"
                    placeholder="footer.message"
                    required=""
                    rows="4"
                  />
                </div>
              </div>
              <button
                class="MuiButtonBase-root-109 MuiButton-root-92 MuiButton-contained-100 MuiButton-fullWidth-108 Hook-buttonSend-1nfwdbi"
                data-testid="footer_submit"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiButton-label-93"
                >
                  footer.send
                </span>
                <span
                  class="MuiTouchRipple-root-126"
                />
              </button>
            </form>
          </div>
          <div
            class="Hook-flexItem-16n2577"
          >
            <p
              class="MuiTypography-root-8 MuiTypography-body2-9 Hook-textHeading-3xpc43"
            >
              footer.questions?
            </p>
            <p
              class="MuiTypography-root-8 MuiTypography-body2-9 Hook-text-i56ocy"
            >
              footer.questionsMessage
            </p>
            <p
              class="MuiTypography-root-8 MuiTypography-body2-9 Hook-text-i56ocy"
            >
              footer.getInTouch
            </p>
            <div
              class="Hook-iconContainer-3qysfj"
            >
              <a
                aria-label="facebook"
                class="MuiButtonBase-root-109 MuiButton-root-92 MuiButton-outlined-97 Hook-button-3hxviu"
                href="hhttps://www.facebook.com/EisbachRiders-262685337908914/"
                rel="noopener"
                role="button"
                tabindex="0"
                target="_blank"
              >
                <span
                  class="MuiButton-label-93"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root-112 Hook-icon-1xdo4e9"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 96.124 96.123"
                  >
                    <g>
                      <path
                        d="M72.089,0.02L59.624,0C45.62,0,36.57,9.285,36.57,23.656v10.907H24.037c-1.083,0-1.96,0.878-1.96,1.961v15.803 c0,1.083,0.878,1.96,1.96,1.96h12.533v39.876c0,1.083,0.877,1.96,1.96,1.96h16.352c1.083,0,1.96-0.878,1.96-1.96V54.287h14.654 c1.083,0,1.96-0.877,1.96-1.96l0.006-15.803c0-0.52-0.207-1.018-0.574-1.386c-0.367-0.368-0.867-0.575-1.387-0.575H56.842v-9.246 c0-4.444,1.059-6.7,6.848-6.7l8.397-0.003c1.082,0,1.959-0.878,1.959-1.96V1.98C74.046,0.899,73.17,0.022,72.089,0.02z"
                      />
                    </g>
                    <g />
                    <g />
                    <g />
                    <g />
                    <g />
                    <g />
                    <g />
                    <g />
                    <g />
                    <g />
                    <g />
                    <g />
                    <g />
                    <g />
                    <g />
                  </svg>
                </span>
              </a>
              <a
                aria-label="instagram"
                class="MuiButtonBase-root-109 MuiButton-root-92 MuiButton-outlined-97 Hook-button-3hxviu"
                href="https://www.instagram.com/eisbachriders/"
                rel="noopener"
                role="button"
                tabindex="0"
                target="_blank"
              >
                <span
                  class="MuiButton-label-93"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root-112 Hook-icon-1xdo4e9"
                    focusable="false"
                    role="presentation"
                    viewBox="0 0 448 512"
                  >
                    <path
                      d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
                    />
                  </svg>
                </span>
              </a>
            </div>
          </div>
        </div>
        <div
          class="Hook-legal-169lkvq"
        >
          <p
            class="MuiTypography-root-8 MuiTypography-body2-9 MuiTypography-alignCenter-24 Hook-copyright-dylql"
          >
            © Eisbach Riders |
             
            <a
              class="MuiTypography-root-8 MuiTypography-colorPrimary-31 MuiLink-root-121 MuiLink-underlineHover-123 Hook-link-13yopjh"
              href="/legal/"
              style="text-decoration: none;"
            >
              footer.imprint
            </a>
            |
             
            <a
              class="MuiTypography-root-8 MuiTypography-colorPrimary-31 MuiLink-root-121 MuiLink-underlineHover-123 Hook-link-13yopjh"
              href="/privacy/"
              style="text-decoration: none;"
            >
              footer.privacy
            </a>
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="Hook-root-4c7lha"
    >
      <div
        class="Hook-container-1tixbb2"
      >
        <div
          class="Hook-flexItem-16n2577"
        >
          <form
            autocomplete="off"
            novalidate=""
          >
            <p
              class="MuiTypography-root-8 MuiTypography-body2-9 Hook-textHeading-3xpc43"
            >
              footer.message
            </p>
            <div
              class="Hook-inputContainer-11ncpjx"
            >
              <div
                class="MuiFormControl-root-39 MuiFormControl-marginNormal-40 MuiTextField-root-38 Hook-textField-xmu9w"
              >
                <label
                  class="MuiFormLabel-root-55 MuiFormLabel-required-60 MuiInputLabel-required-47 MuiInputLabel-root-43 MuiInputLabel-formControl-49 MuiInputLabel-animated-52 Hook-inputLabel-1dirayr"
                  data-shrink="false"
                  for="footer_name"
                >
                  common.name
                  <span
                    class="MuiFormLabel-asterisk-61 MuiInputLabel-asterisk-48"
                  >
                     *
                  </span>
                </label>
                <div
                  class="MuiInputBase-root-75 MuiInput-root-62 MuiInputBase-formControl-76 MuiInput-formControl-63 Hook-input-2r9hv1"
                >
                  <input
                    aria-invalid="false"
                    class="MuiInputBase-input-85 MuiInput-input-70 Hook-textArea-1fn7y44"
                    id="footer_name"
                    placeholder="common.name"
                    required=""
                    type="text"
                    value=""
                  />
                </div>
              </div>
              <div
                class="MuiFormControl-root-39 MuiFormControl-marginNormal-40 MuiTextField-root-38 Hook-textField-xmu9w"
              >
                <label
                  class="MuiFormLabel-root-55 MuiFormLabel-required-60 MuiInputLabel-required-47 MuiInputLabel-root-43 MuiInputLabel-formControl-49 MuiInputLabel-animated-52 Hook-inputLabel-1dirayr"
                  data-shrink="false"
                  for="footer_email"
                >
                  common.email
                  <span
                    class="MuiFormLabel-asterisk-61 MuiInputLabel-asterisk-48"
                  >
                     *
                  </span>
                </label>
                <div
                  class="MuiInputBase-root-75 MuiInput-root-62 MuiInputBase-formControl-76 MuiInput-formControl-63 Hook-input-2r9hv1"
                >
                  <input
                    aria-invalid="false"
                    class="MuiInputBase-input-85 MuiInput-input-70 Hook-textArea-1fn7y44 MuiInputBase-inputType-88 MuiInput-inputType-73"
                    id="footer_email"
                    placeholder="common.email"
                    required=""
                    type="email"
                    value=""
                  />
                </div>
              </div>
            </div>
            <div
              class="MuiFormControl-root-39 MuiFormControl-marginNormal-40 MuiFormControl-fullWidth-42 MuiTextField-root-38"
            >
              <label
                class="MuiFormLabel-root-55 MuiFormLabel-required-60 MuiInputLabel-required-47 MuiInputLabel-root-43 MuiInputLabel-formControl-49 MuiInputLabel-animated-52 Hook-inputLabel-1dirayr"
                data-shrink="false"
                for="footer_message"
              >
                footer.message
                <span
                  class="MuiFormLabel-asterisk-61 MuiInputLabel-asterisk-48"
                >
                   *
                </span>
              </label>
              <div
                class="MuiInputBase-root-75 MuiInput-root-62 MuiInputBase-fullWidth-84 MuiInput-fullWidth-69 MuiInputBase-formControl-76 MuiInput-formControl-63 MuiInputBase-multiline-83 MuiInput-multiline-68 Hook-input-2r9hv1"
              >
                <textarea
                  aria-invalid="false"
                  class="MuiInputBase-input-85 MuiInput-input-70 Hook-textArea-1fn7y44 MuiInputBase-inputMultiline-87 MuiInput-inputMultiline-72"
                  id="footer_message"
                  placeholder="footer.message"
                  required=""
                  rows="4"
                />
              </div>
            </div>
            <button
              class="MuiButtonBase-root-109 MuiButton-root-92 MuiButton-contained-100 MuiButton-fullWidth-108 Hook-buttonSend-1nfwdbi"
              data-testid="footer_submit"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label-93"
              >
                footer.send
              </span>
              <span
                class="MuiTouchRipple-root-126"
              />
            </button>
          </form>
        </div>
        <div
          class="Hook-flexItem-16n2577"
        >
          <p
            class="MuiTypography-root-8 MuiTypography-body2-9 Hook-textHeading-3xpc43"
          >
            footer.questions?
          </p>
          <p
            class="MuiTypography-root-8 MuiTypography-body2-9 Hook-text-i56ocy"
          >
            footer.questionsMessage
          </p>
          <p
            class="MuiTypography-root-8 MuiTypography-body2-9 Hook-text-i56ocy"
          >
            footer.getInTouch
          </p>
          <div
            class="Hook-iconContainer-3qysfj"
          >
            <a
              aria-label="facebook"
              class="MuiButtonBase-root-109 MuiButton-root-92 MuiButton-outlined-97 Hook-button-3hxviu"
              href="hhttps://www.facebook.com/EisbachRiders-262685337908914/"
              rel="noopener"
              role="button"
              tabindex="0"
              target="_blank"
            >
              <span
                class="MuiButton-label-93"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root-112 Hook-icon-1xdo4e9"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 96.124 96.123"
                >
                  <g>
                    <path
                      d="M72.089,0.02L59.624,0C45.62,0,36.57,9.285,36.57,23.656v10.907H24.037c-1.083,0-1.96,0.878-1.96,1.961v15.803 c0,1.083,0.878,1.96,1.96,1.96h12.533v39.876c0,1.083,0.877,1.96,1.96,1.96h16.352c1.083,0,1.96-0.878,1.96-1.96V54.287h14.654 c1.083,0,1.96-0.877,1.96-1.96l0.006-15.803c0-0.52-0.207-1.018-0.574-1.386c-0.367-0.368-0.867-0.575-1.387-0.575H56.842v-9.246 c0-4.444,1.059-6.7,6.848-6.7l8.397-0.003c1.082,0,1.959-0.878,1.959-1.96V1.98C74.046,0.899,73.17,0.022,72.089,0.02z"
                    />
                  </g>
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                  <g />
                </svg>
              </span>
            </a>
            <a
              aria-label="instagram"
              class="MuiButtonBase-root-109 MuiButton-root-92 MuiButton-outlined-97 Hook-button-3hxviu"
              href="https://www.instagram.com/eisbachriders/"
              rel="noopener"
              role="button"
              tabindex="0"
              target="_blank"
            >
              <span
                class="MuiButton-label-93"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root-112 Hook-icon-1xdo4e9"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 448 512"
                >
                  <path
                    d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
                  />
                </svg>
              </span>
            </a>
          </div>
        </div>
      </div>
      <div
        class="Hook-legal-169lkvq"
      >
        <p
          class="MuiTypography-root-8 MuiTypography-body2-9 MuiTypography-alignCenter-24 Hook-copyright-dylql"
        >
          © Eisbach Riders |
           
          <a
            class="MuiTypography-root-8 MuiTypography-colorPrimary-31 MuiLink-root-121 MuiLink-underlineHover-123 Hook-link-13yopjh"
            href="/legal/"
            style="text-decoration: none;"
          >
            footer.imprint
          </a>
          |
           
          <a
            class="MuiTypography-root-8 MuiTypography-colorPrimary-31 MuiLink-root-121 MuiLink-underlineHover-123 Hook-link-13yopjh"
            href="/privacy/"
            style="text-decoration: none;"
          >
            footer.privacy
          </a>
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllBySelectText": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getAllByValue": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getBySelectText": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "getByValue": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllBySelectText": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryAllByValue": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryBySelectText": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "queryByValue": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
